cellom2tif/filetypes.py

```diff
@@ -1,3 +1,32 @@
+import os
+
+
+def fn_has_ext(fn, ext, case_sensitive=False):
```

Comment:
I'd prefer `has_extension`. I was reading recently about API design (I think it was on the Go Language blog), and one of the things they mentioned was to think about how things will read when they are used... In this case, `fn_has_ext(fn, ext)` I don't think reads as well as `has_extension(filename, ext)`. Do you agree?